app/admin/subsite.rb

```diff
@@ -0,0 +1,27 @@
+ActiveAdmin.register Subsite do
+  permit_params :name, :site_id
+
+  index do
+    column :name
+    column :site do |s|
+      "#{s.property&.name} - #{s.name}"
```

Comment:
I like your suggestion to say subsite has one property through site. Based on what we know of the data we're modeling, that feels very safe.